src/bytecode/procs.c

```diff
@@ -1833,6 +1861,7 @@ static BBlock* IR_emit_expr(IR_ProcBuilder* builder, BBlock* bblock, Expr* expr,
         dst->type = expr_str_lit->super.type;
         dst->str_lit = expr_str_lit->str_lit;
 
+        dst->str_lit->used = true;
```

Comment:
Add used string literals into an array (instead of marking them). This also makes it easier (more efficient) to iterate through used string literals.